lib/ansible/modules/packaging/language/npm.py

```diff
@@ -269,6 +280,9 @@ def main():
               unsafe_perm=unsafe_perm, state=state)
 
     changed = False
+    if ci == True:
+        npm.ci_install()
+        changed = True
     if state == 'present':
```

Comment:
Should this be updated to elif when using ci mode?
```suggestion
    elif state == 'present':
```

Reply:
Yes because if CI is used, it will always install so present/absent is not valid with this call.